src/org/yccheok/jstock/gui/JStock.java

```diff
@@ -4425,11 +4445,32 @@ public void mouseReleased(MouseEvent e) {
 
         private void maybeShowPopup(MouseEvent e) {
             if (e.isPopupTrigger()) {
+                if (jTable1.getSelectedRowCount() <= 1) {
+                   SetFocusToRightClickLocation(e, jTable1); 
+                }
                 if (jTable1.getSelectedRowCount() > 0) {
                     getMyJTablePopupMenu().show(e.getComponent(), e.getX(), e.getY());
                 }
             }
         }
+
+        private void SetFocusToRightClickLocation(MouseEvent e, JTable table) {
```

Comment:
I typically do use camelCase. Where did I not case it correctly? When do you typically capitalize the first letter? I would typically only capitalize the first letter for Methods but happy to follow whatever your syntax style is.